cms/models.py

```diff
@@ -117,15 +117,20 @@ def get_context(self, request, *args, **kwargs):
             "js_settings_json": json.dumps(_serialize_js_settings(request)),
             "site_name": settings.SITE_NAME,
             "title": self.title,
-            "program_description_section": self.program_description_section,
         }
 
     @property
-    def alumni(self):
+    def homepage_alumni(self):
```

Comment:
`homepage_` is redundant here. This property is on `Homepage` class so `Homepage.homepage_alumni` seems redundant. Please revert to `Homepage.alumni`.